config/webpack.config.dev.js

```diff
@@ -119,8 +119,18 @@ module.exports = {
       // allow it implicitly so we also enable it.
       {
         test: /\.json$/,
+        exclude: /\/manifest.json$/,
         loader: 'json'
       },
+      // A special case for manifest.json to place it into build root directory.
+      {
+        test: /\/manifest.json$/,
+        include: [paths.appSrc],
```

Comment:
Sorry I wasn't clear. There is no way to make it load differently depending on context. Since we want it to resolve to filename, I wanted to move it away from src folder. The reason is that people normally expect that they can import JS/JSON inside src folder and treat it as a module. So it would be weird if src/manifest.json somehow was treated specially and you couldn't import it as a module from, for example, src/index.js. However we never officially supported importing modules from outside src folder so in this case special behavior seems like fair game. Does this make sense?